Library/Homebrew/cmd/help.rb

```diff
@@ -21,6 +20,7 @@
 
 Further help:
   man brew
+  brew help COMMAND
```

Comment:
And then they run `brew help` and get the exact same output (just a different exit code). But I don't mind adding the square brackets.